.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.8"
+  - "nightly"  # nightly build
+addons:
+  apt:
+    packages:
+    - libsnappy-dev
+# Pycodestyle part
+# needed to work correctly with Python 3 shebang
+env: SKIP_INTERPRETER=true
+install:
+  - pip install pycodestyle
+script:
+  - python run_pycodestyle.py
```

Comment:
Why don't use directly `pycodestile`? In addition, as probably the number of linters will grow, maybe it is better to move pycodestyle to a `requirements-dev.txt` file and just install the dependencies from there.

Reply:
When I checked it, pycodestyle was unable to find all Python sources spread over multiple directories (which is not one project). Probably it has changed?

Reply:
ok now it works, it just does not print total number of errors :)))

Reply:
I usually do `pycodestyle **/*.py` and let bash to find everything, but not sure how would that work on travis.

We had some good tooling around Python code in ccx-data-pipeline repository, using pre-commit and a lot of formatters and linters, but it was like 2 years ago that I don't pay attention to it :-/

Reply:
ok will change it